doc/api/manifest.md

```diff
@@ -252,6 +252,14 @@ _type_: ``Boolean|undefined``
 This property only makes sense for text Adaptations. In this case, if ``true``
 it means that the text track has added hints for the hard of hearing.
 
+#### isTrickModeTrack
+
+_type_ : ``Boolean|undefined``
+
+This property only makes sense for video Adaptations. In this case, if ``true``
+it means that the video track is a trick mode track that will be played only
+if the user start the trick mode on the respective main adaptation.
```

Comment:
I don't understand the sentence :
> it means that the video track is a trick mode track that will be played only
if the user start the trick mode on the respective main adaptation

Isn't it just that we're [already] playing a trickmode track when it is set to `true`?

Reply:
Nope. This property tells if the adaptation is a trick mode.

Reply:
Oh ok, I didn't read poperly where it was written